xlive/H2MOD/Variants/VariantPlayer.cpp

```diff
@@ -57,7 +57,7 @@ DatumIndex VariantPlayer::GetUnitDatum(XUID xuid)
 
 	if (xuid_to_unit_datum.find(xuid) == xuid_to_unit_datum.end())
 	{
-		for (auto i = 0; i < 15; i++)
+		for (auto i = 0; i < 16; i++)
```

Comment:
why is this 16 now?

if this is valid, add a constant for this value, explaining why its "16"

Reply:
commit description clarifies

Reply:
The commit description is "Fixed player loop only going up to index 14 inside VariantPlayer inst……ead of 15 (player 16)-Himanshu01"

That doesn't really explain anything, don't rely on commit descriptions to describe your code. Please make special fields like this a constant with a simple doc.